api/server/onsubmit.js

```diff
@@ -49,5 +49,8 @@ export const callOnSubmit = params => {
     augmentPlayerStageRound(player, player.stage, player.round, game);
   });
 
+  player.stage = _.extend({}, stage);
+  player.round = _.extend({}, round);
+
```

Comment:
Does this require an augmentPlayerStageRound call here? Otherwise calling player.stage.set() will add the value to the root stage, not the player-stage, no?

Reply:
yes it does, my bad, i forgot to add the augmentPlayerStageRound